### PR TITLE
DEVDOCS-4121: [net new] Store Logs API

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-4121](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4121)

New api specs for Store Logs API

This PR is related to [PR 1547](https://github.com/bigcommerce/dev-docs/pull/1547) (api doc) and [PR 118](https://github.com/bigcommerce-labs/next-dev-center-prototype/pull/118) (redirects) 

Ping @bookernath

### CODE REVIEW

File: reference/store_logs.v3.yml [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+              - notification system
+              - email integration
+              - ebay
+              - shopping comparison
+              - order settings
+              - square
+              - facebook
+              - pinterest
+              - design
+              - inventory
```
Comment (bc-andreadao) [inline reference/store_logs.v3.yml:83]: @bookernath Where can I find a list of all the modules? The ones I listed here are from my sandbox store, but was hoping to tell developers all the other types that they can query for. I found some on the UI (screenshot), though I realized the UI display term didn't always match the query term that developers can use.
Type

**Comment by bc-andreadao** [comment]
The commit SHA-5ca1d613075ad9e9dddd760542599281cf5f15e5 changes enum values to example values. It renders fine in Stoplight, but gives an error (see screenshot). Was hoping we could ignore the failed check in Github.

stoplight errors

**Comment by slsriehl** [comment]
@bc-andreadao what's the reason to change `enum` to `examples`? are they not actually [enums](https://swagger.io/docs/specification/data-models/enums/) (a discrete list of all possible/allowed values)? 

If they aren't enums and you want them to appear as pictured, see what happens when you add a `description` property to the schema object itself.

> The commit [5ca1d61](https://github.com/bigcommerce/api-specs/commit/5ca1d613075ad9e9dddd760542599281cf5f15e5) changes enum values to example values. It renders fine in Stoplight, but gives an error (see screenshot). Was hoping we could ignore the failed check in Github.
> 
> stoplight errors

**Review by slsriehl — CHANGES_REQUESTED** [review]
see previous comment. This is close!

**Comment by bc-andreadao** [comment]
It is actually an enum in that it is a discrete list of values (so I changed it back to enum. One thing is that I was not able to get my hands on all possible values, but hopefully that is okay

**Review by stanzikratelbc — COMMENTED** [review]
@bc-andreadao Hi Andrea, I left some comments about updating the descriptions and possibly adding a description for parameters that currently don't have one.

File: reference/store_logs.v3.yml [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+        | --- | --- | --- |
+        |  Store logs | read-only | store_logs_read_only |
+
+        ### Headers
+
+        |Header|Parameter|Description|
+        |-|-|-|
+        |`X-Auth-Token`|`access_token `|Obtained by creating an API account or installing an app in a BigCommerce control panel.|
+
+        * For more information on Authenticating BigCommerce APIs, see: [Authentication](/api-docs/getting-started/authentication).
```
Comment (slsriehl) [inline reference/store_logs.v3.yml:214]:
```suggestion
        * For example request and more information about authenticating BigCommerce APIs, see [Authentication and Example Requests](/api-docs/getting-started/authentication#x-auth-token-header-example-requests).
```

File: reference/store_logs.v3.yml [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+  parameters: {}
+  requestBodies: {}
+  securitySchemes:
+    X-Auth-Token:
+      type: apiKey
+      name: X-Auth-Token
+      in: header
+      description: |-
+        ### OAuth Scopes
+        |  **UI Name** | **Permission** | **Parameter** |
```
Comment (slsriehl) [inline reference/store_logs.v3.yml:204]:
```suggestion
        |  UI Name | Permission | Parameter |
```

File: reference/store_logs.v3.yml [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+        | --- | --- | --- |
```
Comment (slsriehl) [inline reference/store_logs.v3.yml:205]:
```suggestion
        | :-------|:-----------|:----------|
```

File: reference/store_logs.v3.yml [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+        |  Store logs | read-only | store_logs_read_only |
+
+        ### Headers
+
+        |Header|Parameter|Description|
```
Comment (slsriehl) [inline reference/store_logs.v3.yml:210]:
```suggestion
        | Header | Parameter | Description |
```

File: reference/store_logs.v3.yml [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+        |-|-|-|
```
Comment (slsriehl) [inline reference/store_logs.v3.yml:211]:
```suggestion
         |:-------|:----------|:------------|
```

File: reference/store_logs.v3.yml [hunk 7]
```diff
@@ -0,0 +1,218 @@
...
+        |`X-Auth-Token`|`access_token `|Obtained by creating an API account or installing an app in a BigCommerce control panel.|
```
Comment (slsriehl) [inline reference/store_logs.v3.yml:212]:
```suggestion
        | `X-Auth-Token` | `access_token` | Obtained by [creating an API account](/api-docs/getting-started/authentication/rest-api-authentication) or installing an app in a BigCommerce control panel. |
```

**Review by slsriehl — APPROVED** [review]
🎉

**Pull request merged**